app/views/follow_requests/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: follow_request, class: "d-inline-block") do |form| %>
+<%= form_with(model: follow_request, class: "#{dom_id(follow_request.recipient)} d-inline-block", local: false) do |form| %>
```

Comment:
In some places you use remote: true and others local: false. You should try sticking to just one